README.md

```diff
@@ -36,6 +36,14 @@ Wrap your application in the `<Provider>` component so that child components can
 
 All components support [styled-system][styled-system] attributes, allowing you to use `fontSize`, `color`, `px` etc.
 
+## Unstable/Beta components
+
+Some components in the [storybook](https://resin-io-modules.github.io/rendition)
+are marked with a `[beta]` label. These components are experimental and still
+under development. They should not be used in a production application.
```

Comment:
If-you-are-in-the-mood: could also mention that their API might change w/o notice.